client/core/core.go

```diff
@@ -145,6 +145,11 @@ func (c *Core) Run(ctx context.Context) {
 	c.initialize()
 	<-ctx.Done()
 	c.wg.Wait()
+
```

Comment:
This isn't an issue with the present PR, but I don't see anywhere `db.Run(ctx)` is called.  I was looking for it because before `boltDB.Run` returns, it calls `bbolt.DB.Close`, which would have to be done after `Backup` of course.

Reply:
I'm not seeing the full picture for the client startup/shutdown process. I'm not sure why we need to block or a done signal before waiting  when we could wait immediately after initializing and use the done signal to shutdown the processes using the waitgroup, ultimately decrementing it to zero. @buck54321 has a better grasp on the design at play here, I'll leave that to him.

Reply:
That's a bit of a different issue, as I'm more concerned with the db running and closing, but to answer your question, I believe it's because the WaitGroup is not incremented until there are dex connections, so you can't wait until the context is canceled and the goroutines begin shutting down.

Reply:
noted, will take another look.

Reply:
Related: https://github.com/decred/dcrdex/pull/187